config/m4/rdmacm.m4

```diff
@@ -0,0 +1,44 @@
+#
+# Check for RDMACM support
+#
+rdmacm_happy="no"
+AC_ARG_WITH([rdmacm],
+           [AS_HELP_STRING([--with-rdmacm=(DIR)], [Enable the use of RDMACM (default is guess).])],
```

Comment:
Yes, if it's present on the host.
Currently it will only configure ucx with rdmacm, there is no actual usage for it right now in ucx.